301-metadata-repositories.md

```diff
@@ -21,6 +21,8 @@ Note that:
 * A metadata repository MAY be physically distinct from a CNAB registry, or not (e.g., bundles as well as TUF and/or in-toto metadata MAY live as [OCI Artifacts](https://stevelasker.blog/2019/08/25/oci-artifacts-and-a-view-of-the-future/) on a CNAB registry).
 * Both TUF and in-toto are frameworks that can be configured in a wide variety of ways to achieve varying degrees of security. However, we discuss one [minimum viable product (MVP)](#minimum-viable-product-mvp), which has been optimized in particular for [Docker Content Trust / Notary 0.6.1](https://github.com/theupdateframework/notary/releases/tag/v0.6.1), in this document.
 * Although we discuss only the signing and verification of bundles, exactly the same principles apply to images.
+* Trust for different metadata repositories with different roots of trust can be established out-of-band using an approach like [TAP 4](https://github.com/theupdateframework/taps/blob/master/tap4.md). For example, a bundle runtime could be shipped with known good copies of TUF `root` metadata for different repositories, or there could even be a meta metadata repository that distributes these `root` metadata.
+* How to establish whether a `root` metadata file is trustworthy is outside the scope of this document. Simply assuming a `root` metadata file is trustworthy (e.g., by "clicking through") voids all guarantees (of end-to-end authenticity and integrity of bundles) provided by the corresponding metadata repository.
```

Comment:
I would note that this falls outside the scope of this document, but not not necessarily include the last sentence.

Reply:
@glyn Do you agree? In any case, please make a suggestion.

Reply:
I think it's important to be up front about the consequences of failing to establish the necessary trust. I looked into moving the text to 300-CNAB-Security.md, but since it's specific to metadata repositories, it doesn't really fit there.

@radu-matei why do you think the last sentence should be omitted?